builtin/providers/mssql/resource_database.go

```diff
@@ -0,0 +1,122 @@
+package mssql
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceMSsqlDatabase() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceMSsqlDatabaseCreate,
+		Read:   resourceMSsqlDatabaseRead,
+		// Update: resourceMSsqlDatabaseUpdate,
```

Comment:
If `Update` isn't currently necessary it would be better to remove the dead code here (and the function itself as well).